ngraph/MNIST.py

```diff
@@ -0,0 +1,133 @@
+'''
+Copyright 2018 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+'''
+
+
+import tensorflow as tf
```

Comment:
Is it easier to duplicate the mnist code just to use ngraph rather than somehow reuse existing mnist code?

Reply:
I think I'm going to use a different more compelling example than mnist.
See my comment below about whether a jobs component is warranted 
if this example runs a sequence of jobs. 

I'll wait for your comments and then close this PR and resubmit with 
a more compelling example that includes ngraph.